suavestd/api/Transactions.mdx

```diff
@@ -0,0 +1,189 @@
+# Transactions
+
+Transactions is a library with utilities to encode, decode and sign Ethereum transactions.
+
+## Functions
+
+### [encodeRLP](https://github.com/flashbots/suave-std/tree/main/src/Transactions.sol#L109)
```

Comment:
These line numbers are all incorrect - can you fix them with the automatic tool?

I think that the `Structs` section should be moved to the top of this file so that it is clear what the difference between 155 and 1559, as well as transactions and "Request transactions" is.

I think that we should add at least one more line to the top of each file to explain _why_ this library is useful/what often-used function it fulfills for suapp devs.